packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -286,7 +339,7 @@ const Dropdown = ({
         }
     };
 
-    const onKeyPressed = (event, item) => {
+    const onKeyPressed = (event: KeyboardEvent, item: TListItem) => {
```

Comment:
We are passing this function to the **onkeypress** we are not allow to pass `item` to this and because of that we have a ts error here! _again I couldn't find a solution for it :sweat_smile:_